WebRoot/js/ajax/dwt/widgets/DwtListView.js

```diff
@@ -2827,7 +2827,8 @@ function() {
 DwtListView.prototype._sizeChildren =
 function(height) {
 	if (this.headerColCreated && this._listDiv && (height != Dwt.DEFAULT)) {
-		Dwt.setSize(this._listDiv, Dwt.DEFAULT, height - DwtListView.HEADERITEM_HEIGHT);
+		var headerHeight = Dwt.getSize(this._listColDiv).y;
```

Comment:
Predefined height is fixed valued `24px`. And is used as height of all list view in ZWC. And most of the time we don't need change in this as all list view are of same style.

However when we change header style via. css and we add some margin, padding for aesthetic reason, height of header may change and in such case, it should be calculated based on actual height and not predefined value. Else list height calculations gets wrong.

Let me know if you have any question.

Thanks